config/locales/en.yml

```diff
@@ -55,18 +56,20 @@ en:
   auth:
     confirm_password: "Confirm password"
     login: "Login"
-    login_failed: "Username or password is invalid"
     logout: "Logout"
     login_required: "Login required"
     logout_successfully: "You've been logged out"
     password: "Password"
     signup: "Signup"
     signup_successfully: "Account was successfully created"
+    signup_unsuccessfully: "Account was unsuccessfully created"
```

Comment:
this sentence sounds awkward :D How about `Failed to create the account` or sth better ?

Reply:
Fixed in https://github.com/gutakk/ruby-google-scraper/pull/17 as I don't need this anymore.